### PR TITLE
Remove some duplication between the _include_entity() methods of secu…

### LINKED ISSUE #7170: Fix duplicated include_entity() methods for security warnings collectors
See at least, but maybe more:

- [x] `AnchoreJenkinsPluginSecurityWarnings`
- [x] `HarborJSONSecurityWarnings`
- [x] `OpenVASSecurityWarnings`

### PR DESCRIPTION
**Pull request by fniessink**
…rity warning collectors. Closes #7170.

### CODE REVIEW

File: components/collector/src/source_collectors/openvas/security_warnings.py [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+from base_collectors import SecurityWarningsSourceCollector, XMLFileSourceCollector
 from collector_utilities.functions import parse_source_response_xml
 from model import Entities, Entity, SourceResponses
 
 
-class OpenVASSecurityWarnings(XMLFileSourceCollector):
+class OpenVASSecurityWarnings(SecurityWarningsSourceCollector, XMLFileSourceCollector):
     """Collector to get security warnings from OpenVAS."""
 
+    MAKE_ENTITY_SEVERITY_VALUE_LOWER_CASE = True
```
Comment (wkoot) [inline components/collector/src/source_collectors/openvas/security_warnings.py:14]:
```suggestion
    ENTITY_SEVERITY_ATTRIBUTE = "threat"
    MAKE_ENTITY_SEVERITY_VALUE_LOWER_CASE = True
```

File: components/collector/src/source_collectors/bandit/security_warnings.py [hunk 2]
```diff
@@ -28,6 +31,6 @@ def _parse_json(self, json: JSON, filename: str) -> Entities:
 
     def _include_entity(self, entity: Entity) -> bool:
         """Return whether to include the warning in the measurement."""
-        return entity["issue_severity"].lower() in self._parameter("severities") and entity[
-            "issue_confidence"
-        ].lower() in self._parameter("confidence_levels")
+        return super()._include_entity(entity) and entity["issue_confidence"].lower() in self._parameter(
```
Comment (wkoot) [inline components/collector/src/source_collectors/bandit/security_warnings.py]: Is it more readable when moving `super()._include_entity(entity)` to the back of the line, or forcing a linebreak after `and` ?

File: components/collector/src/source_collectors/openvas/security_warnings.py (hunk 1 above)
Comment (fniessink) [reply components/collector/src/source_collectors/openvas/security_warnings.py:14]: That would be wrong. The attribute is called `threat` in the OpenVas XML, but in the Entity it's called `severity`. See the `OpenVASSecurityWarnings` class attribute `ATTRIBUTES` that maps the attribute names from the OpenVAS XML to the entity attribute names.

File: components/collector/src/source_collectors/bandit/security_warnings.py (hunk 2 above)
Comment (fniessink) [reply components/collector/src/source_collectors/bandit/security_warnings.py]: I'll add a helper method for the issue confidence. That way we get something that fits on one line.

File: components/collector/src/source_collectors/openvas/security_warnings.py (hunk 1 above)
Comment (wkoot) [reply components/collector/src/source_collectors/openvas/security_warnings.py:14]: oh! yikes :D

**Pull request merged**